recipes/pepredicates/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "pepredicates" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: de2addf01e2bccbf361f729c21693f3dc56690caec9abfa75c5a118f1e9f4cfd
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - setuptools
+  run:
+    - astropy
+    - matplotlib-base
+    - numpy
+    - python >=3.6
+    - seaborn
+
+test:
+  imports:
+    - pepredicates
+
+about:
+  home: https://git.ligo.org/will-farr/pepredicates
+  doc_url: https://docs.ligo.org/will-farr/pepredicates
+  license: AGPL3
```

Use SPDX name for this
```suggestion
  license: AGPL-3.0
```

---

I have changed this in commit 73d9279